climpred/metrics.py

```diff
@@ -195,24 +339,61 @@ def _less(forecast, reference, dim='svd', comparison=None):
           Prediction System.” Meteorologische Zeitschrift, December 21, 2016,
           631–43. https://doi.org/10/f9jrhw.
 
+    # TODO: not yet fully understood!
+
     Range:
         * pos: under-disperive
         * neg: over-disperive
         * perfect: 0
     """
-    if comparison.__name__ != '_m2r':
-        raise KeyError(
-            'LESS requires member dimension and therefore '
-            "compute_hindcast(comparison='m2r')"
-        )
-    numerator = _mse(forecast, reference, dim='member').mean(dim)
+    dim2 = 'init'  # [i for i in forecast.dims if i != 'member']
+    if 'init' in forecast.coords:
+        dim2 = 'init'
+    elif 'time' in forecast.coords:
+        dim2 = 'time'
+
+    forecast, ref2 = xr.broadcast(forecast, reference)
+    numerator = _mse(forecast, ref2, dim='member').mean(dim2)
     # not corrected for conditional bias yet
-    denominator = _mse(forecast.mean('member'), reference.mean('member'), dim=dim)
+    denominator = _mse(forecast.mean('member'), ref2.mean('member'), dim=dim2)
     less = np.log(numerator / denominator)
     return less
 
 
-def _bias(forecast, reference, dim='svd', comparison=None):
+def _crpss_es(forecast, reference, **kwargs):
+    """CRPSS Ensemble Spread.
+
+    .. math:: CRPSS = 1 - \\frac{CRPS(\\sigma^2_f)}{CRPS(\\sigma^2_o}))
+
+    References:
+        * Kadow, Christopher, Sebastian Illing, Oliver Kunst, Henning W. Rust,
+          Holger Pohlmann, Wolfgang A. Müller, and Ulrich Cubasch. “Evaluation
+          of Forecasts by Accuracy and Spread in the MiKlip Decadal Climate
+          Prediction System.” Meteorologische Zeitschrift, December 21, 2016,
+          631–43. https://doi.org/10/f9jrhw.
+
+    # TODO: not yet fully understood!
+
+    Range:
+        * perfect: 0
+    """
+    rdim = [tdim for tdim in reference.dims if tdim in climpred_dims + ['time']]
+    dim2 = 'init'
```

Comment:
don't think this is necessary if it's going to be redefined in the if statements right below